custom/css/style.css

```diff
@@ -771,31 +771,53 @@ padding-bottom:80px;
 .pycon2018_panel{
 
     width: 95%;
-padding:10px 1px;
+    /*padding:10px 1px;*/
     margin: 0 auto;
 
     box-shadow: 0px 3px 10px 0px #ccc;
 
 }
 
-.pycon2018_panel .table>thead>tr>th{
-    border-bottom:none;
-    color:#e23e89;
-    font-weight:600;
-     font-family: 'Roboto', sans-serif;
-
-}
 .pycon2018_panel .table>tbody>tr>td{
     border-top:none;
      font-family: 'Roboto', sans-serif;
      font-size:15px;
      height:100px;
 }
+
+.pycon2018_panel .row {
+  border-top:none;
+     font-family: 'Roboto', sans-serif;
+     font-size:15px;
+}
+
+.pycon2018_panel > .row:nth-of-type(even) {
+    background: #f9f9f9;
+}
+
 .pycon2018_panel .table>tbody>tr>td.time, td .time{
     font-weight:bold;
      color:#e23e89;
 }
 
+.pycon2018_panel .time{
+    font-weight:bold;
+    color:#e23e89;
+}
+
+.ticket-column {
+  position: relative;
+  height: 130px;
```

Comment:
use `em` instead of `px`. will make responsive according to the screen size

Reply:
@vaibhavsingh97 
Tried using 10em for desktop, but looks too spacious for mobile, anything less then that messes with vertical alignment of buttons.
Attached screen-shot for 7em, vertical alignment of buttons still looks out of place. 
Do I have to use media query with `em` too? e.g `10em` on desktop and `5em` on mobile?
![screen shot 2018-06-29 at 10 43 54 pm](https://user-images.githubusercontent.com/11061604/42105737-1d893ef2-7bef-11e8-81e3-74ede439dd20.png)

value  |  desktop | Mobile
------ | --------- | --------
10em | 👍 | 👎 
5em | 👎  | 👍

Reply:
Ideally we should use `em` for everything to make everything responsive. But let's just use `px` in this case as it looks ok in both mobile and desktop views.